metabase/metabase
# Convert `headings.module.css` to CSS modules

### Description

Converts `headings.module.css` to CSS modules. This include the `h1` to `h6` selectors, and a few media queries for changing the heading  size on smaller screens.

### How to verify

Check the files, and ensure that the css module selectors match the old class strings.
